### PR TITLE
Removed cdn_url property from all the layers

### LINKED ISSUE #3983: viz.json includes cdn_url
it should not send cdn_url for any layer

### PR DESCRIPTION
**Pull request by ethervoid**
Removed `cdn_url` property from `viz.json` because this info now comes from the tile and is not necessary here for compatibility reasons.

After this change is commited I assume that a `bundle exec rake cartodb:redis:purge_vizjson` task should be launched to remove old vizs from cache.

**This PR fixes #3983**

/cc @javisantana

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4716/))

**Comment by javisantana** [comment]
I'd like to test if an old cartodb.js version would work without this, taking a look to old cartodb.js versions cc @alonsogarciapablo

**Comment by javisantana** [comment]
looks like our latests supported cartodb.js version: 3.11.24. It's available since 3.8, so we are safe here.

There is something that will be broken: dynamic_cdn flag. 

That flag allows request to be sent through the CDN instead of going directly to the tiler, it's used when we are under high load. That flag does not make any sense anymore since backend could change ``maps_api_template`` to reach CDN instead our servers directly so I'd remove it too.

see the cartodb.js issue https://github.com/CartoDB/cartodb.js/issues/564

**Comment by javisantana** [comment]
also, remember that in order to drop a database column we need to:

- remove all the usages in the code
- merge and deploy
- create a migration that removes the column
- merge and deploy

if we do the code change and migration to remove the column at the same time the requests will fail while it being deployed to all the servers

**Comment by ethervoid** [comment]
I will clean the `dynamic_cdn` property too and merge this change and deploy. If everything goes smooth, I will create a migration file to remove that column and will deploy it :)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4776/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4792/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4793/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4796/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4817/))

**Comment by ethervoid** [comment]
Need code review because I've removed the `dynamic_cdn` property too. I want to deploy this today if possible

// @CartoDB/backend

File: app/controllers/carto/api/vizjson_presenter.rb [hunk 1]
```diff
@@ -28,7 +28,6 @@ def calculate_vizjson(options={})
           user_api_key: user.api_key,
           user: user,
           viewer_user: user,
```
Comment (Kartones) [inline app/controllers/carto/api/vizjson_presenter.rb]: trailing commas at arrays are for cowards and bad PHP developers! :fire:

File: app/models/visualization/member.rb [hunk 2]
```diff
@@ -639,7 +639,6 @@ def calculate_vizjson(options={})
           user_api_key: user.api_key,
           user: user,
           viewer_user: user,
```
Comment (Kartones) [inline app/models/visualization/member.rb]: :fire:  to the trailing comma

**Comment by Kartones** [comment]
:+1: after you remove those commas

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4839/))

**Pull request merged**